common/selectors/selectAndDedupe.js

```diff
@@ -13,14 +13,12 @@ const dedupe = require("lib/dedupe");
 module.exports = function selectAndDedupe(group) {
   return group.reduce((result, options, index, arr) => {
     let current;
+    let sites = options.defaults ? options.sites.concat(options.defaults) : options.sites;
```

Comment:
That's correct. Thanks for clarifying it! @Mardak 

As `group.defaults` is of the array type, I think we can replace the ternary
`let sites = options.defaults ? options.sites.concat(options.defaults) : options.sites;`
with
`let sites = options.sites.concat(options.defaults);`

Also, looks like `group.dedupe` is an unused argument. Let's update the doc for the better readability.